docs/stdlib/array.rst

```diff
@@ -161,14 +159,13 @@ Reference
 
 .. eql:operator:: arrayslice: array<anytype> [ int64 : int64 ] -> anytype
 
-    Array slicing.
-
-    An omitted lower bound defaults to zero, and an omitted upper
-    bound defaults to the size of the array.
+    Slices an array of :eql:type:`anytype` containing :eql:type:`int64`.
 
-    The upper bound is non-inclusive.
+    This results in a representable reference of the array's elements.
 
-    Examples:
+    Omitting the lower bound an array will default the result to zero.
```

Comment:
>... will default the result to zero.
This reads as though the result of the slice will default to zero when actually the low bound will default to zero.

Suggestion:
> ... will result in a default low bound of zero.

Reply:
```suggestion
    Omitting the lower bound of an array slice will default to a low bound of
    zero.
```